### PR TITLE
i18n(es): Update `cloudflare.mdx`

### PR DESCRIPTION
**Pull request by K1ri86**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change?
- Translated and Update #4933

### I am participating in the Hacktoberfest!

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 1]
```diff
@@ -226,7 +226,14 @@ export default defineConfig({
 
 ## Runtime de Cloudflare
 
-Te proporciona acceso a [variables de entorno](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables).
+Te proporciona acceso a [variables de entorno](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables), y a las [vinculaciones de Cloudflare](https://developers.cloudflare.com/pages/platform/functions/bindings).
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:229]:
```suggestion
Te proporciona acceso a las [variables de entorno](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables) y [bindings de Cloudflare](https://developers.cloudflare.com/pages/platform/functions/bindings).
```

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 2]
```diff
@@ -226,7 +226,14 @@ export default defineConfig({
...
+
+Las vinculaciones actualmente admitidas son:
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:231]:
```suggestion
Los bindings actualmente admitidos son:
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**